forge-gui/res/cardsfolder/upcoming/racketeer_boss.txt

```diff
@@ -0,0 +1,14 @@
+Name:Racketeer Boss
```

Comment:
Do you mind removing this file from this PR?

I think the new tweak is going to require a tiny engine tweak and I will open up a separate PR for it. I will use your script as a starting point though! Shouldn't have to change much.